### ISSUE TITLE
hashtag autocomplete accessibility

### ISSUE THREAD

**Issue by ryanflorence** [michaelbromley/angularUtils#3]
currently the hashtag autocomplete directive isn't accessible to screenreaders, here are some resources:

http://www.w3.org/TR/wai-aria-practices/#autocomplete
http://www.w3.org/TR/wai-aria/states_and_properties#aria-autocomplete
http://oaa-accessibility.org/example/10/

**Comment by michaelbromley** [comment]
Closing this as I am unable to further develop that module (plus I don't think it is much used).
